auslib/test/admin/views/test_emergency_shutoff.py

```diff
@@ -0,0 +1,176 @@
+        dbo.emergencyShutoff.scheduled_changes.t.insert().execute(
+            sc_id=1, scheduled_by="bill", change_type="update",
+            data_version=1, base_shutoff_id=3, base_product='Firefox',
+            base_channel='test', base_updates_disabled=False,
+            base_data_version=1)
+        dbo.emergencyShutoff.scheduled_changes.conditions.t.insert().execute(
+            sc_id=1, data_version=1, when=1000000)
+        dbo.productRequiredSignoffs.t.insert().execute(
+            product="Thunderbird", channel="release", role="releng", signoffs_required=1, data_version=1)
+        dbo.productRequiredSignoffs.t.insert().execute(
+            product="Firefox", channel="release", role="releng", signoffs_required=1, data_version=1)
+
+    def test_get_emergency_shutoff_list(self):
+        resp = self._get('/emergency_shutoff')
+        self.assertStatusCode(resp, 200)
+        data = json.loads(resp.data)
+        self.assertEquals(data['count'], 3)
+        self.assertIn('shutoffs', data)
+
+    def test_get_emergency_shutoff(self):
+        resp = self._get('/emergency_shutoff/1')
+        self.assertStatusCode(resp, 200)
+        self.assertIn('X-Data-Version', resp.headers)
+        data = json.loads(resp.data)
+        self.assertIn('product', data)
+        self.assertEquals(data['product'], 'Firefox')
+        self.assertIn('channel', data)
+        self.assertEquals(data['channel'], 'nightly')
+
+    def test_get_emergency_shutoff_notfound(self):
+        resp = self._get('/emergency_shutoff/404')
+        self.assertStatusCode(resp, 404)
+
+    def test_get_filtered_emergency_shutoffs(self):
+        qs = {'product': 'Fennec'}
+        resp = self._get('/emergency_shutoff', qs=qs)
+        self.assertStatusCode(resp, 200)
+        data = json.loads(resp.data)
+        for shutoff in data['shutoffs']:
+            self.assertEquals('Fennec', shutoff['product'])
+
+        qs = {'channel': 'nightly'}
+        resp = self._get('/emergency_shutoff', qs=qs)
+        self.assertStatusCode(resp, 200)
+        data = json.loads(resp.data)
+        for shutoff in data['shutoffs']:
+            self.assertEquals('nightly', shutoff['channel'])
+
+        qs = {'channel': 'test', 'product': 'Firefox'}
+        resp = self._get('/emergency_shutoff', qs=qs)
+        self.assertStatusCode(resp, 200)
+        data = json.loads(resp.data)
+        for shutoff in data['shutoffs']:
+            self.assertEquals('Firefox', shutoff['product'])
+            self.assertEquals('test', shutoff['channel'])
+
+    def test_create_emergency_shutoff(self):
+        data = {'product': 'Thunderbird',
+                'channel': 'release',
+                'additional_notification_list': 'm@m.org'}
+        resp = self._post('/emergency_shutoff', data=data.copy())
+        self.assertStatusCode(resp, 201)
+
+        qs = {'channel': 'release', 'product': 'Thunderbird'}
+        resp = self._get('/emergency_shutoff', qs=qs)
```

Comment:
You're testing the _post() function in this test, but this test will start to fail if _get() breaks. It would be better to pull the new values directly from the database (like we do in other web tests) instead of relying on the HTTP API for them.